docs/vi/week01/ch01.md

```diff
@@ -0,0 +1,22 @@
+---
+lang: vi
+lang-ref: ch01.md
+title: week01
```

Comment:
Translate `week 01` to Vietnamese.

Reply:
Can I use this format?

Reply:
You're supposed to translate "Week 1" to Vietnamese.

Reply:
> You're supposed to translate "Week 1" to Vietnamese.

I will use the same your format